auraed/src/logging/logchannel.rs

```diff
@@ -60,28 +59,24 @@ impl LogChannel {
 
     /// Getter for consumer channel
     pub fn get_consumer(&self) -> Receiver<LogItem> {
-        self.consumer.clone()
+        self.producer.subscribe()
     }
 
     /// Wrapper that sends a log line to the channel
     pub fn log_line(producer: Sender<LogItem>, line: &str) {
-        match producer.send(LogItem {
+        // send returns an Err if there are no receivers. We ignore that.
+        let _ = producer.send(LogItem {
             channel: "unknown".to_string(),
             line: line.to_string(),
             // TODO: milliseconds type in protobuf requires 128bit type
             timestamp: get_timestamp_sec(),
-        }) {
-            Ok(_) => {}
-            Err(e) => {
```

Comment:
are we losing some error logging with this change?

Reply:
Yes and no. LogChannel will only send messages while there is a receiver (connected gRPC client). Clients only get messages that are produced after they connect/subscribe. So if there is no receiver, LogChannel will lose messages.  However, SimpleLogger is also wired up, and that will receive all log messages I believe.